test-case/multiple-pause-resume.sh

```diff
@@ -135,6 +136,27 @@ expect {
         if { \$i > $repeat_count } { exit 0 }
         exp_continue
     }
+    -re "#.*MAX" {
+        if { \$maxcount > 10 } { puts "\rMAX Volume error!"; exit 1 }
+        puts "\rError: MAX volume detected (\$maxcount/10), Wait for 500ms"
+        incr maxcount
```

Comment:
Is it possible to reach MAX 10 times but just like "MAX -> xx% -> MAX -> ...". What I mean is: should `maxcount` be reset in other cases?

Reply:
That's good point. Will do that.
But if pause/resume don't care about volume, I need to merge this case with normal volume case.